### DISCUSSION TITLE
How to solve the problem of not supporting Chinese search?

### DISCUSSION THREAD

**Discussion by fu1996** [afconsult/quill-mention#408] [Q&A]
After cloning the code library, modifying the case, adding Chinese options, and using @ to search, it was found that English is acceptable, but Chinese is not.

**Comment by Beamon-project** [comment]
![image](https://github.com/user-attachments/assets/b0d3813b-8d7b-4b11-9718-7da2add01034)

**Reply by badspider7** [reply]
allowedChars: /^[A-Za-z\u4e00-\u9fa5\s]*$/, // 修改这里以支持中文
